### DISCUSSION TITLE
根据文章封面的存在情况，调整卡片样式

### DISCUSSION THREAD

**Discussion by CheckingChen** [MaLuns/hexo-theme-async#65] [Ideas 奇思妙想]
比如说一个文章是没有设置封面的，能否做到在index页中隐藏默认封面图，实现如下效果呢？
![1](https://user-images.githubusercontent.com/64892457/220023336-2e776700-dd6c-42fa-ac6d-d95f4ce3834d.png)
1、2皆为无封面的文章，正常显示标题、分类等信息，单栏列表同理

**Comment by MaLuns** [comment]
使用瀑布流去排版 ？

**Comment by CheckingChen** [comment]
唔...原本想法是直接对齐，瀑布流也可行，不过排出来会不会有些乱？

**Comment by MaLuns** [comment]
用瀑布流，排版应该像这样，无封面图卡片不是偶数情况下，底部会不对齐

![image](https://user-images.githubusercontent.com/31614024/220224146-8a10a6c2-831d-40bf-bbc4-d7c64530b476.png)

**Reply by CheckingChen** [reply]
这个一直都是瀑布流的通病吧，也只能这样了😢

**Reply by MaLuns** [reply]
> 这个一直都是瀑布流的通病吧，也只能这样了😢

可以用个卡片填充空白地方，比如广告位？或者其他的🤣🤣🤣

**Reply by CheckingChen** [reply]
试试用虚线框填充？里面的内容还可以自定义样式（实现广告位招租的效果🙃）
不过无封面内容是偶数时会凸出来一块吗？或者说偶数可以让它占两个卡片格，像单栏那样展示

**Comment by JingxuGu** [comment]
感觉做成瀑布流这样子确实很好，避免单调性的同时省去了很多找头图的头疼，有些文章是真找不到什么相关的头图🤣
